### PR TITLE
Change document for REC

### LINKED ISSUE #1246: Documenting superseded and obsoleted properly
Today, we have:
* https://www.w3.org/pubrules/doc/rules/?profile=REC-SUPERSEDED  
* https://www.w3.org/pubrules/doc/rules/?profile=REC-OBSOLETE

but we don't following those.

Instead we do
  https://pad.w3.org/p/superseding-and-obsoleting-a-rec

Is there a way to update the documentation?

### PR DESCRIPTION
**Pull request by jennyliang220**
resolves #1246

### CODE REVIEW

**Comment by jennyliang220** [comment]
Hi @plehegar , see previews at attachment: [document-preview.zip](https://github.com/w3c/specberus/files/6590183/document-preview.zip)

**Review by plehegar — APPROVED** [review]

**Comment by plehegar** [comment]
LGTM. thank you @jennyliang220

File: lib/rules.json [hunk 1]
```diff
@@ -1575,102 +1575,46 @@
...
-                            "namespaces": true,
-                            "wcag": true,
-                            "fixupJs": true
+                            "previous": ["obsoleted"]
                         }
                     },
-                    "compound": {
-                        "name": "8. Compound Documents",
+                    "rec-edit-sotd": {
+                        "name": "6. Change \"status of this document\" section",
```
Comment (deniak) [inline lib/rules.json:1609]:
```suggestion
                        "name": "6. Change the \"Status of this Document\" section",
```

File: lib/rules.json [hunk 2]
```diff
@@ -1679,102 +1623,46 @@
...
-                            "lastStylesheet": true,
-                            "viewport": true,
-                            "canonical": true
+                            "install" : true
                         }
                     },
-                    "front-matter": {
-                        "name": "3. Front Matter",
+                    "rec-edit-stylesheet": {
+                        "name": "2. Use corresponding stylesheet",
```
Comment (deniak) [inline lib/rules.json:1633]:
```suggestion
                        "name": "2. Use the corresponding stylesheet",
```

File: lib/rules.json [hunk 3]
```diff
@@ -1679,102 +1623,46 @@
...
                         "rules": {
-                            "toc": true,
-                            "tocNav": true
+                            "h2": ["superseded"]
                         }
                     },
-                    "document-status": {
-                        "name": "5. Document Status Section",
+                     "rec-edit-version": {
+                        "name": "4. Set This version link",
```
Comment (deniak) [inline lib/rules.json:1645]:
```suggestion
                        "name": "4. Set the \"This version\" link",
```

File: lib/rules.json [hunk 4]
```diff
@@ -1679,102 +1623,46 @@
...
-                            "knownDisclosureNumber": true,
-                            "whichProcess": true
+                            "version": ["SPSD"]
                         }
                     },
-                    "document-body": {
-                        "name": "7. Document Body",
+                     "rec-edit-previous": {
+                        "name": "5. Set Previous version link",
```
Comment (deniak) [inline lib/rules.json:1651]:
```suggestion
                        "name": "5. Set the \"Previous version\" link",
```

File: lib/rules.json [hunk 5]
```diff
@@ -1679,102 +1623,46 @@
...
-                            "namespaces": true,
-                            "wcag": true,
-                            "fixupJs": true
+                            "previous": ["superseded"]
                         }
                     },
-                    "compound": {
-                        "name": "8. Compound Documents",
+                    "rec-edit-sotd": {
+                        "name": "6. Change \"status of this document\" section",
```
Comment (deniak) [inline lib/rules.json:1657]:
```suggestion
                        "name": "6. Change the \"Status of this Document\" section",
```

File: lib/rules.json [hunk 6]
```diff
@@ -1872,6 +1760,48 @@
...
                 }
+            },
+            "rec-edit-install": {
+                "name": "1. Install the document.",
+                "rules": {
+                    "install":"(if new dated version) Copy the original REC in /TR into a new dated <code>&lt;new date&gt;</code> directory in /TR"
+                }
+            },
+            "rec-edit-stylesheet": {
+                "name": "2. Use corresponding stylesheet",
```
Comment (deniak) [inline lib/rules.json:1771]:
```suggestion
                "name": "2. Use the corresponding stylesheet",
```

File: lib/rules.json [hunk 7]
```diff
@@ -1872,6 +1760,48 @@
...
+                "rules": {
+                    "stylesheet":"For the W3C style sheet, use the @{param1} version of the W3C style sheet in use in the REC (old, or 2016). So, it's one of:<ol><li>https://www.w3.org/StyleSheets/TR/W3C-@{param1}.css</li><li>https://www.w3.org/StyleSheets/TR/2016/W3C-@{param1}.css</li><li>or reuse the (some?) content from https://www.w3.org/TR/html50/superseded.css as needed</li></ol>"
```
Comment (deniak) [inline lib/rules.json:1773]:
```suggestion
                    "stylesheet":"For the W3C stylesheet, use the @{param1} version of the W3C stylesheet in use in the REC (old, or 2016). So, it's one of:<ol><li>https://www.w3.org/StyleSheets/TR/W3C-@{param1}.css</li><li>https://www.w3.org/StyleSheets/TR/2016/W3C-@{param1}.css</li><li>or reuse the (some?) content from https://www.w3.org/TR/html50/superseded.css as needed</li></ol>"
```

File: lib/rules.json [hunk 8]
```diff
@@ -1872,6 +1760,48 @@
...
+                }
+            },
+            "rec-edit-h2": {
+                "name": "3. Change the title of the document",
+                "rules": {
+                    "h2":"In the <code>&lt;h2&gt;</code> element containing the state and date (\"W3C Recommendation DD MMMMMM YYYY\"), add:<ol><li>a <code>&lt;br&gt;</code> element after the original date</li><li>and \"@{param1} <code>&lt;new date&gt;</code>\" when the @{param1} document gets published by the webmaster</li></ol>"
+                }
+            },
+            "rec-edit-version": {
+                "name": "4. Set This version link",
```
Comment (deniak) [inline lib/rules.json:1783]:
```suggestion
                "name": "4. Set the \"This version\" link",
```

File: lib/rules.json [hunk 9]
```diff
@@ -1872,6 +1760,48 @@
...
+                "rules": {
+                    "version":"(if new dated version) This version link gets the form https://www.w3.org/TR/YYYY/@{param1}-&lt;shortname-with-version&gt;-&lt;newdate&gt;/"
+                }
+            },
+            "rec-edit-previous": {
+                "name": "5. Set Previous version link",
```
Comment (deniak) [inline lib/rules.json:1789]:
```suggestion
                "name": "5. Set the \"Previous version\" link",
```

File: lib/rules.json [hunk 10]
```diff
@@ -1872,6 +1760,48 @@
...
+                "rules": {
+                    "previous": "(if new dated version) Previous version becomes the dated link of the REC being @{param1}"
+                }
+            },
+            "rec-edit-sotd": {
+                "name": "6. Change \"status of this document\" section",
```
Comment (deniak) [inline lib/rules.json:1795]:
```suggestion
                "name": "6. Change the \"status of this document\" section",
```

**Review by deniak — CHANGES_REQUESTED** [review]
The list for the section 6 (Change "status of this document" section) doesn't seem right.
6.1, 6.2, etc should be in a sublist like in section 3.

Also, `/doc/rules/?profile=REC-OBSOLETE` mentions "For an example, see the diff between the REC and the Obsoleted document of High Resolution Time Level 1", but hr-time-1 is a superseded REC.
Screen Shot 2021-06-04 at 09 30 21

**Review by deniak — APPROVED** [review]

**Pull request merged**